recipes/bdwgc/all/patches/update-cmake.patch

```diff
@@ -0,0 +1,834 @@
+--- CMakeLists.txt
```

Comment:
many changes, did you take this patch somewhere or created by hand?
what's the summary of the changes?
consider renaming to something more specific that "update cmake" or adding comment(s)

Reply:
This is the current master's cmake script.